CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## unreleased
 
+* Venmo
+  * Make `VenmoRequest` parcelable
```

Comment:
Question - is this ok as a non-breaking change? The only issue I could see is if a merchant is subclassing `VenmoRequest`, their subclass will now also have to implement `Parcelable`. There is no need for a merchant to subclass this, so not sure if we think this is ok.

Reply:
This is needed because `VenmoRequest` is a property on `DropInRequest` in `v6`, which is also `Parcelable`. I think it makes sense to treat this breaking change as `non-breaking`. 

The number of merchants subclassing this type _should_ be low. I think the value added to the `DropIn` API outweighs any risk of a breaking change occurring for a non-standard integration.

Reply:
Agreed, I think we're good to count this as non breaking!